src/components/_button.scss

```diff
@@ -180,6 +219,21 @@
   font-size: $fz-l;
 }
 
+@mixin sui-icon--small {
```

Comment:
Ok, could be a mixin for that, but using variables makes icons more flexible/scalable, the same way we're doing for button properties.